tasks.py

```diff
@@ -123,8 +123,49 @@ def get_first_lineno_with_prefix(text: str, prefix: str) -> int:
 
 
 @task
+def type_stubs(ctx):
```

Comment:
Just moving this above the `type_check` task so we can reference the dependency the "correct" invoke way (I was hoping it would do caching like Make, but apparently not).